glacierscript.py

```diff
@@ -884,5 +968,10 @@ def withdraw_interactive():
     if args.program == "create-withdrawal-data":
         withdraw_interactive()
 
+    if args.program == "sign-transaction":
+        # Sign an existing transaction (i.e. add a signature to partially signed tx) to withdaw funds from cold storage
+        RE_SIGN_MODE = 1
+        withdraw_interactive()
```

Comment:
Related to my comment above (about extracting that block into a separate function), perhaps it would be cleaner to have two methods, e.g withdraw_new() and withdraw_continue(), implemented separately, with the common parts pulled out into functions.